packages/application-generic/package.json

```diff
@@ -62,6 +62,7 @@
     "@novu/gupshup": "^0.20.0-alpha.0",
     "@novu/infobip": "^0.20.0-alpha.0",
     "@novu/kannel": "^0.20.0-alpha.0",
+    "@novu/mailchain" : "0.16.3",
```

Comment:
This needs to be the same version as all of the other providers

Reply:
Why is every version the same, if I may ask?